### DISCUSSION TITLE
Traceback to determine plugin(s) that throw warning regarding update to Confuse

### DISCUSSION THREAD

**Discussion by j-peeters** [beetbox/beets#4024] [Ideas]
With the new 1.5 update Beet shows the following warning for my specific setup. Likely due to a plugin that is using the deprecated beets.util.confit instead of Confuse.  
```
/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/beets/util/confit.py:21: UserWarning: beets.util.confit is deprecated; use confuse instead   warnings.warn("beets.util.confit is deprecated; use confuse instead")
```
It would be great if Beets would give a traceback to show from which plugin this error is coming so that a improvement request can be made to the plugins developer.

**Comment by wisp3rwind** [comment]
We could use https://stackoverflow.com/a/22376126/3451198 to achieve that.

**Comment by wisp3rwind** [comment]
Hopefully, https://github.com/beetbox/beets/pull/4263 solves this.
